css/styles.css

```diff
@@ -1,58 +1,51 @@
 /* буквы */
 p {
-    display: inline-block;
-    width: 100px;
-    text-align: center;
-    margin: 6px -1px;
-    font-size: 30px;
+  display: inline-block;
+  width: 100px;
+  text-align: center;
+  margin: 6px -1px;
```

Comment:
why we used margin, not padding?

Reply:
Because padding cannot have a negative value ?)